tests/functional/modules/test_zos_lineinfile_func.py

```diff
@@ -103,7 +103,7 @@
 ZOAU_ROOT=/usr/lpp/zoautil/v100
 export ZOAU_ROOT
 export _BPXK_AUTOCVT
-export ZOAU_ROOT"""
```

Comment:
The change it was because the test before add a line already in the text to the end of it, so just adding the ' to be different

Reply:
I did a local test to check the behavior of `ansible.builtin.lineinfile` against my localhost (MacOS). I used the file:

ABC.txt:
```
AA
BB
CC
```

And the playbook task:
```
  - name: Run builtin lineinfile locally
    ansible.builtin.lineinfile:
      path: "{{ playbook_dir }}/files/ABC.txt"
      insertafter: EOF
      line: BB
    delegate_to: localhost
```

And found that the module find the existing `BB` line and does not append another `BB` to the EOF. `zos_lineinfile` behavior appears to be consistent. This also explains why the change to the line is needed. It would not be possible to add another line `export ZOAU_ROOT` using zos_lineinfile if one already existed elsewhere in the file.